packages/admin-app/src/components/JointKeyWizard/Steps/GuardianAssignmentStep.tsx

```diff
@@ -90,92 +85,66 @@ const GuardianAssignmentStep: React.FC<GuardianAssignmentStepProps> = ({
 
     const queryClient = new QueryClient();
 
+    const initialGuardians = baseJointKey.guardians.map((g) => g.id);
+    const userClient = useUserClient();
+    const skip = 0;
+    const take = 100;
```

Comment:
Is this pagination?

Reply:
The guardian API needs to have the skip and take passed into it and it would be used for pagination.

Reply:
Correct, pagination